libraries/AP_GPS/AP_GPS_UBLOX.h

```diff
@@ -590,6 +646,7 @@ class AP_GPS_UBLOX : public AP_GPS_Backend
         STEP_POLL_GNSS, // poll GNSS
         STEP_POLL_TP5, // poll TP5
         STEP_TMODE, // set TMODE-MODE
+        STEP_RTK_MOVBASE, // setup moving baseline
```

Comment:
This should be after `STEP_VERSION` otherwise you will always have to rerun the first 13 steps for no gain, which means that you take 9 seconds longer to do the setup then needed.

Reply:
done